src/promptflow-devkit/promptflow/_proxy/_python_inspector_proxy.py

```diff
@@ -36,6 +37,9 @@ def get_entry_meta(
     ) -> Dict[str, Any]:
         timeout = kwargs.get("timeout", FLOW_META_JSON_GEN_TIMEOUT)
         load_in_subprocess = kwargs.get("load_in_subprocess", True)
+        # if it's from eval batch run, use the same process to generate flow meta for sake of performance
+        if is_eval_batch_run():
```

Comment:
Can we pass the `load_in_process` parameter from evaluation entry?

Reply:
Open new process to gen meta to avoid influencing `sys.path` and `working dir` in other threads.
Although it could be set from kwargs, seems like we don't have usage to set it as false for now.

https://github.com/search?q=repo%3Amicrosoft%2Fpromptflow%20get_entry_meta&type=code

Reply:
One option I'm considering is to remove is_eval_batch_run() and use a new environment variable, PF_LOAD_FLOW_META_IN_SUBPROCESS, to control the behavior. What do you think?

Reply:
Moving this change to a seperate PR.